core/src/main/java/org/mule/runtime/core/internal/routing/ScatterGatherRouter.java

```diff
@@ -95,4 +98,18 @@ protected ForkJoinStrategyFactory getDefaultForkJoinStrategyFactory() {
   private boolean isLazyInit() {
     return configurationProperties.resolveBooleanProperty(MULE_LAZY_INIT_DEPLOYMENT_PROPERTY).orElse(false);
   }
+
+  /**
+   * Validates that the payload is not consumable so it can be copied.
+   *
+   * If validation fails then throws a MessagingException
+   *
+   * @param message
+   * @throws MuleException if the payload is consumable
+   */
+  public static void validateMessageIsNotConsumable(Message message) {
```

Comment:
is this change related?

Reply:
ScatterGather was using that method from one of the removed classes (yeah, it depended on a class of first successful 🤦‍♂ )